torch/distributed/tensor/parallel/ddp.py

```diff
@@ -0,0 +1,90 @@
+from typing import Any, List, Tuple
+
+import torch.nn as nn
+from torch.distributed.tensor.parallel.fsdp import _flatten_tensor, _unflatten_tensor
```

Comment:
Odd place to import from, but can be changed later.

Reply:
We can make this to a util.

Reply:
Separate these funcs in https://github.com/pytorch/pytorch/pull/107313/files.